### PR TITLE
Enable cashbox by default

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Cashbox should always be enabled](https://trello.com/c/WhUb2bzK/559-cashbox-should-always-be-enabled)

**Comment by chaotixkilla** [comment]
[Reject expired IDs](https://trello.com/c/TXOi8uy4/565-reject-expired-ids)

**Review by josepfo — APPROVED** [review]

File: new-lamassu-admin/src/components/editableTable/Row.js [hunk 1]
```diff
@@ -192,7 +192,7 @@ const ERow = ({ editing, disabled, lastOfGroup }) => {
 
   const classes = useStyles()
 
-  const shouldStripe = stripeWhen && stripeWhen(values) && !editing
+  const shouldStripe = stripeWhen && stripeWhen(values)
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/components/editableTable/Row.js:195]: Have you make sure that other places that contain stripe aren't being broken by this change?

File: new-lamassu-admin/src/pages/Maintenance/CashCassettes.js [hunk 2]
```diff
@@ -159,6 +159,8 @@ const CashCassettes = () => {
     }
   ]
 
+  const isAnyStripeFalse = () => !!R.find(R.propEq('stripe', false))(elements)
+
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Maintenance/CashCassettes.js]: Why do we need this?

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: new-lamassu-admin/src/pages/Maintenance/CashCassettes.js (hunk 2 above)
Comment (chaotixkilla) [reply new-lamassu-admin/src/pages/Maintenance/CashCassettes.js]: We don't actually. Fixed

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**